test/data.jl

```diff
@@ -54,7 +54,13 @@ module TestData
     @test size(df6, 2) == 3
 
     #test_group("null handling")
-    @test nrow(df5[complete_cases(df5), :]) == 3
+    @test nrow(df5[completecases(df5), :]) == 3
+    @test nrow(dropnull(df5)) == 3
+    returned_copy = dropnull(df4)
+    @test df4 == returned_copy && !(df4 === returned_copy)
+    @test nrow(dropnull!(df5)) == 3
+    returned_view = dropnull!(df4)
```

Comment:
This is the same test concept we just merged into NullableArrays https://github.com/JuliaStats/NullableArrays.jl/blob/master/test/primitives.jl#L208

If `deleteat!` receives no indices to remove, I thought it would just be a return of the same data. Is view the incorrect term for that in this case?